config.ini

```diff
@@ -109,7 +108,7 @@ webhook = url
 ;type = content
 ;# Mode of the filter.
 ;# "regexp" requires "regexp" property and should contain... A RegExp
-;# "hashtag" should contain space-separated list of tags
+;# "tag" should contain space-separated list of tags
```

Comment:
I feel like at one point I wanted `hashtag` to be a synonym for `tag` in config, but probably forgot about actually adding it; having it as `tag` makes more sense anyways